src/AtelierTomato.Markov.Service.Discord/DiscordObjectOIDBuilder.cs

```diff
@@ -5,19 +5,30 @@ namespace AtelierTomato.Markov.Service.Discord
 {
 	public class DiscordObjectOIDBuilder
 	{
-		public static async Task<DiscordObjectOID> Build(IGuild? guild, IChannel channel, ulong messageID, string instance = "discord.com")
+		/// <summary>
+		/// Builds a DiscordObjectOID at scope of <see cref="DiscordObjectOID.Channel"/> or <see cref="DiscordObjectOID.Thread"/>.
+		/// </summary>
+		/// <param name="guild">The <see cref="IGuild"/> of the desired DiscordObjectOID. Must contain the <paramref name="channel"/>.</param>

```

Comment:
do a remarks tag or something to says that this uses the api and thus incurs big cost
builders usually do not do that.